payments-core/src/main/java/com/stripe/android/model/ElementsSession.kt

```diff
@@ -60,8 +60,43 @@ data class ElementsSession(
             val liveMode: Boolean,
             val apiKey: String,
             val apiKeyExpiry: Int,
-            val customerId: String
+            val customerId: String,
+            val components: Components,
         ) : StripeModel
+
+        @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+        @Parcelize
+        data class Components(
+            val paymentSheet: PaymentSheetComponent,
+            val customerSheet: CustomerSheetComponent,
```

Comment:
This naming will cause some Dagger-related confusion 😅

Reply:
Could simplify it to just `PaymentSheet` and `CustomerSheet` but that could also cause confusion 😞. Any names you think could be better?

Reply:
Let’s use your suggestion. Should be fine with the containing classes.